client/my-sites/upgrades/checkout-thank-you/index.jsx

```diff
@@ -199,15 +201,18 @@ const CheckoutThankYou = React.createClass( {
 					primaryPurchase={ primaryPurchase }
 					selectedSite={ this.props.selectedSite } />
 
-				<div className="checkout-thank-you__features-header">
-					{ this.translate( "Get started with your site's new features" ) }
-				</div>
-
-				<div className="checkout-thank-you__purchase-details-list">
-					<ComponentClass
-						selectedSite={ selectedSite }
-						domain={ domain } />
-				</div>
+				<CheckoutThankYouFeaturesHeader
+					isDataLoaded={ this.isDataLoaded() }
```

Comment:
We could, but it feels to me like that could get missed if we ever refactored this. If we keep it as a function call then we are more confident that it will contain the correct value in any context.